### DISCUSSION TITLE
[ZK Stack] - error(prover_fri): Error occurred while processing FriCpuProver job: "unsatisfied"

### DISCUSSION THREAD

**Discussion by SuccinctPaul** [zkSync-Community-Hub/zksync-developers#310] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [X] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.74.0-nightly

### Operating System

- [X] Ubuntu
- [ ] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

See more: https://github.com/matter-labs/zksync-era/issues/1008

### Expected Behavior

See more: https://github.com/matter-labs/zksync-era/issues/1008

### Current Behavior

See more: https://github.com/matter-labs/zksync-era/issues/1008

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by Mixaakisev** [comment]
oyy thx

**Comment by BHUPESH200** [comment]
Good

**Comment by SuccinctPaul** [comment, marked as answer]
Bugfixed with new code
